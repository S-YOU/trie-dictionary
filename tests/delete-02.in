find aba
insert aaaaa
find aba
insert aabaa
find aba
insert abaaa
find aba
insert abbaa
find aba
delete 0
find aba
delete 1
find aba
delete 2
find aba
delete 3
find aba
delete 4
find aba
delete 31337
find aba
delete 987654
delete 33554431
