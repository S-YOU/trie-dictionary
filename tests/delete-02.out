NO
word number: 0
NO
word number: 1
NO
word number: 2
YES
word number: 3
YES
deleted: 0
YES
deleted: 1
YES
deleted: 2
NO
deleted: 3
NO
ignored
NO
ignored
NO
ignored
ignored
